packages/cli/src/Generate.ts

```diff
@@ -50,6 +50,7 @@ ${chalk.bold('Options')}
       --schema   Custom path to your Prisma schema
   --data-proxy   Enable the Data Proxy in the Prisma Client
        --watch   Watch the Prisma schema and rerun after a change
+   --generator   Specifies which generator to use to generate assets (may be provided multiple times)
```

Comment:
Here it's probably best to shorten the description a bit more. How about something like this?

```suggestion
   --generator   Generator to use (may be provided multiple times)
```